src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+const sortList = (listOfValue) => {
+  const listWithNumber = [...listOfValue];
+
+  function normalizeNumber() {
+    for (const item of listWithNumber) {
+      item.dataset.salary = item.dataset.salary.split(',').join('').slice(1);
+    }
+  };
+
+  normalizeNumber();
```

Comment:
It's better to make function just to convert string, and then use it in loop on each value. This vay it's more flexible and better corresponds to its name
```suggestion
  function normalizeNumber(numberString) {
    return numberString.split(',').join('').slice(1);
  };

  for (const item of listWithNumber) {
      item.dataset.salary = normalizeNumber(item.dataset.salary);
  }
```